docs/REST API/rest-api.md

```diff
@@ -9483,10 +9542,12 @@ Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
 | self_service | boolean | form | Whether this is optional self-service software that can be installed by the end user. |
 | labels_include_any        | array     | form | Target hosts that have any label in the array. Only one of either `labels_include_any` or `labels_exclude_any` can be specified. |
 | labels_exclude_any | array | form | Target hosts that don't have any label in the array. |
+| automatic_install | boolean | form | Create a policy that triggers a software install only on hosts missing the software. |
+| automatic_update | boolean | form | Create a policy that triggers a software install on all hosts running an older version. |
```

Comment:
```suggestion
| automatic_install | boolean | form | Trigger a software install only on hosts missing the software. |
| automatic_update | boolean | form | Trigger a software install on all hosts running an older version. |
```

Thinking "create a policy" language doesn't apply here because we'll either create, modify, or delete the associaetd policy as needed based on bool value and when the package updates.

As a heads-up, we'll likely have to revise naming conventions on this sooner rather than later to differentiate vs. "automatically update _the version of the FMA in the library"